src/commands/hset.json

```diff
@@ -41,7 +41,20 @@
             }
         ],
         "reply_schema": {
-            "type": "integer"
+            "oneOf": [
+                {
+                    "description": "The field/fields is/are new field/fields in the hash and value/values was/were set correspondingly.",
+                    "type": "integer"
+                },
+                {
+                    "description": "The field/fields already exists in the hash and new value/values were set correspondingly.",
+                    "const": 0
+                },
+                {
+                    "description": "The field/fields are combination of new and already exists in the hash and new value/values were set correspondingly.",
+                    "type": "integer"
+                }
+            ]
```

Comment:
there is only one case: "The number of new fields"
why did you split it into 3 cases?

Reply:
Soory Guybe, I address the simple way: The number of new fields that were added, Please take a look.